Testscripts/Linux/perf_apache.sh

```diff
@@ -0,0 +1,255 @@
+#!/bin/bash
+
+#######################################################################
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+#######################################################################
+
+#######################################################################
+#
+# perf_apache.sh
+# Description:
+#    Install apache and run apache performance tests.
+#    This script needs to be run on client VM.
+#
+# Supported Distros:
+#    Ubuntu/Suse/Centos/RedHat/Debian
+#######################################################################
+
+CONSTANTS_FILE="./constants.sh"
+UTIL_FILE="./utils.sh"
+
+. ${CONSTANTS_FILE} || {
+	errMsg="Error: missing ${CONSTANTS_FILE} file"
+	LogErr "${errMsg}"
+	SetTestStateAborted
+	exit 10
+}
+. ${UTIL_FILE} || {
+	errMsg="Error: missing ${UTIL_FILE} file"
+	LogErr "${errMsg}"
+	SetTestStateAborted
+	exit 10
+}
+
+if [ ! "${server}" ]; then
+	errMsg="Please add/provide value for server in constants.sh. server=<server ip>"
+	LogErr "${errMsg}"
+	SetTestStateAborted
+	exit 1
```

Comment:
Same as above

Reply:
? If the test is aborted, what number we should exit with?

Reply:
As per the design, testscript should always exit with 0. Set the correct state before exiting. In this case SetTestStateAborted is correctly set.

Reply:
@johnsongeorge-w  Thank you for telling me this. I have fixed.